CMakeLists.txt

```diff
@@ -321,8 +323,18 @@ if(${HAVE_BABELTRACE})
   set(WITH_BABELTRACE ${BABELTRACE_FOUND})
   set(HAVE_BABELTRACE_BABELTRACE_H ${BABELTRACE_FOUND})
   set(HAVE_BABELTRACE_CTF_EVENTS_H ${BABELTRACE_FOUND})
+  set(HAVE_BABELTRACE_CTF_ITERATOR_H ${BABELTRACE_FOUND})
 endif(${HAVE_BABELTRACE})
 
+option(DEBUG_GATHER "C_Gather debugging is enabled" OFF)
+option(HAVE_KINETIC "Kinetic Client is enables" OFF)
```

Comment:
can get rid of it, along with the kinetic backend code - it's really just of historical interest in it's current form